### ISSUE TITLE
Yokogawa cell voyager "actions" failure

### ISSUE THREAD

**Issue by scottberry** [TissueMAPS/TissueMAPS#59]
My dataset contains multiple image files with different "action" but the same "channel name". e.g.

```
..._C11_T0001F001L01A01Z01C01.tif
..._C11_T0001F001L01A01Z01C02.tif
..._C11_T0001F001L01A02Z01C01.tif
..._C11_T0001F001L01A02Z01C02.tif
```

The `A01` and `A02` should be treated as different channels but currently cause failure at metaconfig

```
2017-08-17 13:31:33 | WARNING  | tmlib.workflow.metaconfig.cellvoyager    | no microscope metadata files found
2017-08-17 13:34:33 | WARNING  | tmlib.workflow.metaconfig.base           | OMEXML does not specify a Plate element
2017-08-17 13:34:33 | WARNING  | tmlib.workflow.metaconfig.api            | required metadata information is missing: "well"
2017-08-17 13:34:35 | WARNING  | tmlib.workflow.metaconfig.api            | microscope stage positions are not available: "Stage position information is not available."

JOB FAILED:
Each well must have the same number of acquisition sites.
  File "/home/tissuemaps/tmlibrary/tmlib/workflow/cli.py", line 283, in __main__
    cli_instance(arguments)
  File "/home/tissuemaps/tmlibrary/tmlib/workflow/cli.py", line 342, in __call__
    method(**method_args)
  File "/home/tissuemaps/tmlibrary/tmlib/workflow/cli.py", line 400, in run
    api.run_job(batch, assume_clean_state)
  File "/home/tissuemaps/tmlibrary/tmlib/workflow/metaconfig/api.py", line 216, in run_job
    stitch_dimensions=(batch['n_vertical'], batch['n_horizontal'])
  File "/home/tissuemaps/tmlibrary/tmlib/workflow/metaconfig/base.py", line 553, in determine_grid_coordinates_from_layout
    'Each well must have the same number of acquisition sites.'
```

**Comment by sparkvilla** [comment]
The problem here concerns the REGEX defined in cellvoyager.py that does not distinguish between two files having same **channel** number BUT different **action** number. I have changed the code in the base.py file so that it checks for the **action** field within the filename.  I also changed the REGEX in cellvoyager.py  now returning "C01" in place of "1" when parsing channel 1. The final channel field will be equal to "A01_C01" or "A02_C01", i.e. action and channel. This should more comprehensible then 1_1 or 2_1 from the user perspective.
